runtime/lua/nvim/health.lua

```diff
@@ -121,30 +112,33 @@ local function check_config()
   local writeable = true
   local shadaopt = vim.fn.split(vim.o.shada, ',')
   local shadafile = (
-    empty(vim.o.shada) and vim.o.shada
+    vim.fn.empty(vim.o.shada) == 1 and vim.o.shada
```

Comment:
when `vim.fn.empty(str)` is used to check for an empty string, we should instead just compare `vim.o.shada == ''`